fairing/training/native/deployment.py

```diff
@@ -26,7 +26,7 @@ class NativeDeployment(object):
             will generate multiple jobs.
     """
 
-    def __init__(self, namespace, runs, output):
+    def __init__(self, namespace, runs, output=None):
```

Comment:
The append build was blowing up because it called super without providing output.  The native deployment checks if `self.outputs` exists and doesn't use it if it doesn't.